bin/next

```diff
@@ -12,7 +12,11 @@ if (pkg.peerDependencies) {
       // When 'npm link' is used it checks the clone location. Not the project.
       require.resolve(dependency)
     } catch (err) {
-      console.warn(`${dependency} not found. Please install ${dependency} using 'npm install ${dependency}'`)
+      if (dependency === 'react' || dependency === 'react-dom') {
```

Comment:
I think we don't need to check this only for react and react-dom. Just make this message the default.